proxy/metacd_http.c

```diff
@@ -238,7 +238,9 @@ handler_action (struct http_request_s *rq, struct http_reply_ctx_s *rp)
 		if (oio_url_has(args.url, OIOURL_VERSION) && !oio_str_is_number(
 					oio_url_get(args.url, OIOURL_VERSION), &args.version))
 			rc = _reply_format_error(&args, BADREQ("Invalid version"));
-		else {
```

Comment:
You could also check the NS (if present) is managed by the current `oio-proxy`, as in https://github.com/open-io/oio-sds/blob/master/proxy/lb_actions.c#L229